### PR TITLE
feat: Implemented an endpoint for retrieving all languages

### PR DESCRIPTION
**Pull request by abdullai-t**
####  Summary / Highlights

This pull request introduces the following changes:

- Introduced a new `languages.json` file in the `src/database/raw_data/other` directory, containing various foreign language options along with their respective country codes.

- An endpoint for fetching all languages that a super admin can add as supported languages for translation on the platform.

#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [x] I've given my PR a meaningful title.
* [x] I linked this PR to the relevant issue.
* [x] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

File: src/api/handlers/translation.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+class TranslationsHandler(RouteHandler):
+	def __init__(self):
+		super().__init__()
+		
+		self.service = TranslationsService()
+		self.registerRoutes()
+	
+	def registerRoutes(self):
+		self.add("/translations.languages.get", self.get_all_languages)
```
Comment (Opoku-Agyemang) [inline src/api/handlers/translation.py:16]:
```suggestion
		self.add("/translations.supported_languages.list", self.list_all_supported_languages)
```

File: src/api/services/translation.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class TranslationsService:
+	"""
+	Service Layer for all the translations
+	"""
+	
+	def __init__(self):
+		self.store = TranslationsStore()
+	
+	def get_all_languages(self, context, args) -> (dict, Exception):
```
Comment (Opoku-Agyemang) [inline src/api/services/translation.py:13]:
```suggestion
	def list_all_supported_languages(self, context, args) -> (dict, Exception):
```

**Review by Opoku-Agyemang — CHANGES_REQUESTED** [review]

File: src/api/handlers/translation.py (hunk 1 above)
Comment (abdullai-t) [reply src/api/handlers/translation.py:16]: This endpoint is meant to get all languages(from a json) that an admin can add to the database as supported languages. @archx3 will create an endpoint for fetching all currently supported languages from the database.

Comment (Opoku-Agyemang) [reply src/api/handlers/translation.py:16]: if you are getting a listed of languages, the api should end with list.

File: src/database/raw_data/other/languages.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+"en_US" : "English (US)",
+"es_ES" : "Spanish (Spain)",
+"pt_BR" : "Brazilian Portuguese",
+"fr_FR" : "French (France)",
+"de_DE" : "German (Germany)",
+"ja_JP" : "Japanese (Japan)",
+"zh_CN" : "Chinese Simplified (China)",
+"zh_TW" : "Chinese Traditional (Taiwan)",
+"it_IT" : "Italian (Italy)"
```
Comment (Opoku-Agyemang) [inline src/database/raw_data/other/languages.json:10]: why not indented?>=

**Review by Opoku-Agyemang — CHANGES_REQUESTED** [review]

**Review by Opoku-Agyemang — APPROVED** [review]

**Pull request merged**